src/tls/certificate.rs

```diff
@@ -38,7 +38,7 @@ pub(crate) fn make_cert(keypair: &Keypair) -> Result<rcgen::Certificate, super::
     // The libp2p-specific extension to the certificate contains a signature of the public key
     // of the certificate using the libp2p private key.
     let libp2p_ext_signature = {
-        let certif_pubkey = certif_keypair.public_key_raw();
+        let certif_pubkey = certif_keypair.public_key_der();
```

Comment:
If this is the SubjectPublicKeyInfo, then this change is correct.

Reply:
Yes it is the SubjectPublicKeyInfo.